### PR TITLE
Generate initial OLM Working Group docs

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -28,5 +28,29 @@ workinggroups:
   contact:
     slack: kubernetes-operators
     mailing_list: https://groups.google.com/forum/#!forum/operator-framework-wg-ci
+- dir: wg-olm
+  name: OLM
+  mission_statement: >
+    To organize community input for the Operator-Lifecycle-Manager project.
```
Comment (kevinrizza) [inline sigs.yaml:34]: Should we include the operator registry and opm in this as well?

Comment (awgreene) [reply sigs.yaml]: That's a good idea.

File: sigs.yaml [hunk 2]
```diff
@@ -28,5 +28,30 @@ workinggroups:
   contact:
     slack: kubernetes-operators
     mailing_list: https://groups.google.com/forum/#!forum/operator-framework-wg-ci
+- dir: wg-olm
+  name: OLM
+  mission_statement: >
+    To organize community input for the Operator Lifecycle Manager project and it's
```
Comment (kevinrizza) [inline sigs.yaml:34]: nit `its` is the possessive, `it's` is a conjunction. Yay English! :)

File: wg-olm/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# OLM Working Group
+
+To organize community input for the Operator Lifecycle Manager project and it's integration with other Operator Framework projects.
```
Comment (kevinrizza) [inline wg-olm/README.md:11]: Same nit as above

Comment (awgreene) [reply wg-olm/README.md]: Resolved in the latest version of the PR.

File: sigs.yaml (hunk 2 above)
Comment (awgreene) [reply sigs.yaml]: Resolved in the latest version of the PR.

File: sigs.yaml (hunk 1 above)
Comment (awgreene) [reply sigs.yaml]: Resolved in the latest version of the PR.

**Pull request merged**